mne/export/_eeglab.py

```diff
@@ -42,7 +42,7 @@ def _export_epochs(fname, epochs):
     ch_names = [ch for ch in epochs.ch_names if ch not in drop_chs]
     cart_coords = _get_als_coords_from_chs(epochs.info['chs'], drop_chs)
 
-    if len(epochs.annotations) > 0:
+    if (epochs.annotations is not None) and (len(epochs.annotations) > 0):
```

Comment:
Do we just need to check for False-y, then?

```suggestion
    if epochs.annotations:
```